astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py

```diff
@@ -1,8 +1,8 @@
 import os
+from datetime import datetime
```

Comment:
Is this change required?

Reply:
mypy is throwing an error is we use airflow path for datatime